### ISSUE TITLE
Malformed URL in OpenAPI UI when leading "/" is missing in JAX-RS for 18.0.0.3

### ISSUE THREAD

**Issue by rieckpil** [OpenLiberty/open-liberty#5421]
Hi,

I get a malforemd URL in the Open API UI at `http://localhost:9080/openapi/ui/` running a simple Java EE 8.0 and Microprofile 2.0.1. application when I am not adding the leading "/" to my JAX-RS `@Path("sample")` annotation:

```
@ApplicationPath("resources")
public class JAXRSConfiguration extends Application {

}

@Path("sample")
public class SampleResource {

	@Inject
	@ConfigProperty(name = "message")
	private String message;

	@GET
	public Response message() {
		return Response.ok(message).build();
	}

}
```

![image](https://user-images.githubusercontent.com/14176609/47106561-334d1200-d247-11e8-8986-045e2111270b.png)

with this malformed URL in the OPen API UI I get a Http 404 when I try to access the endpoint from the UI. Accessing the URL in the browser `http://localhost:9080/liberty-vpot-october/resources/sample`  works as expected. 

Adding the "/" to `@Path("/sample")` results in the correct URL format in the Open API UI:

![image](https://user-images.githubusercontent.com/14176609/47106716-a5255b80-d247-11e8-9c74-8d7b7baf0c92.png)

The Java doc for the `@Path("")` says the following regarding the leading "/":

>  ... For the purposes of absolutizing a path against the base URI , a leading '/' in a path is ignored and base URIs are treated as if they ended in '/' ...

**Comment by arturdzm** [comment]
Thank you for reporting the issue.
I was able to replicate this.

**Comment by yeekangc** [comment]
@arturdzm, do you have an update for this issue?  Thanks.

**Comment by gcharters** [comment]
@arturdzm is there an update on this issue.  I have come across a number of REST services that have bumped into this.  Thanks.

**Comment by leochr** [comment]
Fixed by PR #6245
